### DISCUSSION TITLE
Exemplary ReqIF files for ERTMS Subset-26

### DISCUSSION THREAD

**Discussion by fkromer** [strictdoc-project/reqif#42] [Q&A] [answered]
For [ERTMS Subset-26](https://www.era.europa.eu/) (a standard for train control systems) one can request example ReqIF files [here](https://www.reqif.academy/template/reqif-demo-file-etcs/). It seems like it's sufficient to register as a free member.

**Comment by stanislaw** [comment, marked as answer]
This file is already included with some fixes in StrictDoc's integration tests. There, being quite a big file it slows down the integration tests quite a bit, so I was going to integrate it parts of it here.

**Reply by stanislaw** [reply]
see here: https://github.com/strictdoc-project/strictdoc/tree/main/tests/integration/reqif/fmStudio/90_SUBSET-026-3-v330

**Reply by stanislaw** [reply]
The parser for this quite opinionated document has been removed from StrictDoc with the following rationale:

```
It turns out that the implementation of this parser might be completely
outdated because the ERTMS Subset-26 documents were generated by a very
old version of FmStudio.
```

See https://github.com/strictdoc-project/strictdoc/pull/612
